docs/asciidoc/metrics.adoc

```diff
@@ -96,17 +97,11 @@ Interested in "events per second" metric? Measure the rate of `reactor.onNext.de
 
 Want to be alerted when the listener throws an error? `reactor.flow.duration` with `status=error` tag is your friend.
 
-=== Common tags
-
-Every metric will have the following tags in common:
-[width="100%",options="header"]
-|=======
-| tag name | description | example
+Please note that when giving a name to a sequence, this sequence could not be aggregated with others anymore. As a compromise if you want to identify your sequence but still make it possible to aggregate with other views, you can use a <<Custom tags>> for the name by calling `(tag("flow", "events"))` for example.
 
-| type | Publisher's type | "Mono"
+=== Common tag
```

Comment:
I wonder if, given the size of this section, it couldn't be merged with the `Custom Tags` section below.